react/__snapshots__/examples.spec.jsx.snap

```diff
@@ -8,7 +8,7 @@ exports[`ActionMenu should render examples: ActionMenu 1`] = `
     <div>
       <div style=\\"position: absolute; left: 0px; top: 0px; z-index: #fff; transform: translate(0px, 0px);\\" data-popper-reference-hidden=\\"true\\" data-popper-escaped=\\"true\\" data-popper-placement=\\"bottom-start\\">
         <div class=\\"styles__c-actionmenu___22Fp1 styles__c-actionmenu--inline___1SXZa\\">
-          <div class=\\"styles__media___cSJMp styles__media--top___K9w0I styles__c-actionmenu-item___gODqd\\">
+          <div class=\\"styles__media___cSJMp undefined styles__c-actionmenu-item___gODqd\\">
```

Comment:
Careful, I do not think that setting align to "middle" is allowed, this is why there are "undefined" classes now.

Reply:
Wow, I read it the other way around when I reread it...

For the 'middle' case, according to the doc https://docs.cozy.io/cozy-ui/react/#/Content?id=media it's supposed to work, right ? (see `align=middle (default)`)

Reply:
fixed here https://github.com/cozy/cozy-ui/pull/1571